scss/exercisePreview.module.scss

```diff
@@ -1,10 +1,14 @@
 @use 'colors.module.scss';
 
-.exercise__options {
+.exercise__answer {
   max-height: 500px;
 }
 
-.exercise__pre__bg {
+pre {
```

Comment:
I don't think NextJS will allow this global selector. It didn't complain (console or terminal)? 

If it did, It should be a class.

Reply:
Oh yeah, i actually had it removed, it was just for testing purposes, not sure why it got included.